recipe/meta.yaml

```diff
@@ -71,6 +84,37 @@ outputs:
         - ./omp_hello            # [unix]
         - '%cd%\omp_hello.exe'   # [win]
 
+  - name: llvm-openmp-fortran
+    script: install_pkg.sh   # [unix]
+    script: install_pkg.bat  # [win]
+    build:
+      # restricted to windows for now
+      skip: true  # [not win]
+      run_exports:
+        strong:
+          - {{ pin_subpackage("llvm-openmp", max_pin=None) }}
+          - _openmp_mutex >={{ openmp_ver }}  # [linux]
+          - _openmp_mutex * *_llvm            # [linux]
+    requirements:
+      build:
+        - cmake
+        # for run-exports
+        - {{ stdlib("c") }}
+        - {{ compiler("cxx") }}
+        - {{ compiler("fortran") }}     # [linux]
+      host:
+        - {{ pin_subpackage("llvm-openmp", exact=True) }}
+      run:
+        - {{ pin_subpackage("llvm-openmp", exact=True) }}
+      run_constrained:
```

Comment:
We need a run_constrained on `flang` here as the mod file made here might only work with the flang compiler version used.